src/Components/Chef/_Chef.scss

```diff
@@ -52,6 +60,30 @@
     width: 478px;
     height: 372px;
   }
+
+  @include mobile{
```

Comment:
if you decice to insert include mobile to every element separately, stick with it. this is an example of mixed approaches.